src/sql-parser/src/ast/defs/value.rs

```diff
@@ -238,3 +239,211 @@ impl Default for IntervalValue {
         }
     }
 }
+
+pub enum BytesUnit {
+    B,
+    KB,
+    MB,
+    GB,
+    TB,
+    PB,
```

Comment:
Double check me on this, but I believe Rust capitalization rules for acronyms yield "Kb", "Gb" etc.

```suggestion
    B,
    Kb,
    Mb,
    Gb,
    Tb,
    Pb,
```

Reply:
Yes, Rust does camel case for acronyms. In this case I was not sure since `Gb` actually means something different than `GB`